src/lwc/geFormWidgetTokenizeCard/__tests__/geFormWidgetTokenizeCard.test.js

```diff
@@ -2,6 +2,15 @@ import { createElement } from 'lwc';
 import GeFormWidgetTokenizeCard from 'c/geFormWidgetTokenizeCard';
 import { DISABLE_TOKENIZE_WIDGET_EVENT_NAME } from 'c/geConstants';
 import { fireEvent } from 'c/pubsubNoPageRef';
+import { getRecord } from 'lightning/uiRecordApi';
+import { getObjectInfo } from 'lightning/uiObjectInfoApi';
+import { registerLdsTestWireAdapter } from '@salesforce/sfdx-lwc-jest';
+
+const mockGetRecord = require('./data/DIMockRecord.json');
+const mockObjectInfo = require('./data/dataImportObjectDescribeInfo.json');
+
+const getRecordAdapter = registerLdsTestWireAdapter(getRecord);
```

Comment:
Both getRecordAdapter and getObjectInfoAdapter are no longer needed as of sfdx-lwc-jest 0.12, can delete these two lines.